issue_processing/jira_issue_processing.py

```diff
@@ -36,15 +37,19 @@
 
 from csv_writer import csv_writer
 
+from jira import JIRA
+
 reload(sys)
 sys.setdefaultencoding('utf-8')
 
+
 def run():
     # get all needed paths and argument for the method call.
     parser = argparse.ArgumentParser(prog='codeface', description='Codeface extraction')
     parser.add_argument('-c', '--config', help="Codeface configuration file", default='codeface.conf')
     parser.add_argument('-p', '--project', help="Project configuration file", required=True)
     parser.add_argument('resdir', help="Directory to store analysis results in")
+    parser.add_argument('-s', '--skip_history', help='Skips methods that retrieve the history', action='store_true')
```

Comment:
I would like to add some more context information in the help text here what "retrieve" means exactly. I propose something like this: "Skip methods that retrieve additional history information from the configured JIRA server".